packages/glyphs-block/src/settings.ts

```diff
@@ -0,0 +1,89 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MultiInputLayout } from '@frontify/fondue';
+import { Choice, appendUnit, defineSettings, numericalOrPixelRule } from '@frontify/guideline-blocks-settings';
+import { getBorderRadiusSettings, getBorderSettings } from '@frontify/guideline-blocks-shared';
+import { getFontWeights, getFonts } from './helpers';
+
+export const DEFAULT_CHARS = 'A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,1,2,3,4,5,6,7,8,9';
+
+const HAS_BACKGROUND_ID = 'hasBackground';
+const FONT_FAMILY_ID = 'fontFamily';
+const FONT_SIZE_ID = 'fontSize';
+
+export const settings = defineSettings({
+    basics: [
+        {
+            id: FONT_FAMILY_ID,
+            defaultValue: 'default',
+            type: 'dropdown',
+            info: 'Select a font family from the project.',
+            choices: async ({ getAppBridge }): Promise<Choice[]> => {
+                const appBridge = getAppBridge();
+                const fonts = await getFonts(appBridge.getProjectId());
+                fonts.unshift({ label: 'Default', value: 'inherit' });
+                return fonts;
+            },
+        },
+        {
+            id: 'chars',
+            type: 'textarea',
+            label: 'Characters',
+            defaultValue: DEFAULT_CHARS,
+            placeholder: 'e.g. A,B,C',
+            info: 'Add characters with a comma between them.',
+        },
+    ],
+    style: [
+        {
+            id: 'font',
+            type: 'multiInput',
+            label: 'Font selection',
+            lastItemFullWidth: true,
+            blocks: [
+                {
+                    id: 'fontWeight',
+                    type: 'dropdown',
+                    defaultValue: 'normal',
```

Comment:
I don't see anything by changing from 400 to normal with my font. Probably just my font. still wanted to check. Does it work with one of yours?

Reply:
This is because `400 = normal`. It depends on the font which font weights are available.

Reply:
makes sense 👍